website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+---
```

Comment:
What about (Alpha) to be an sub-menu? E.g 

scorecard
|----- stable
|----- alpha (--alpha)

Reply:
I was going to just remove the older scorecard when the alpha version moves from alpha, but other scorecard related docs would be sub-menu docs (e.g. custom image writing).

Reply:
yep, I saw other PR for other related docs as well :-) 
see that it still: 

<img width="257" alt="Screenshot 2020-05-21 at 00 24 33" src="https://user-images.githubusercontent.com/7708031/82507222-9c6aca80-9af9-11ea-9aeb-ee903ffe008a.png">

To change we need to create the dir and add both files inside.

Reply:
reorganized to have the scorecard dir with the contents under it.